src/uwp_events.h

```diff
@@ -0,0 +1,119 @@
+/**********************************************************************************************
+*
+*   raylib.uwp_events - Functions for bootstrapping UWP functionality within raylib's core.
+*
+*
+*   LICENSE: zlib/libpng
+*
+*   Copyright (c) 2014-2020 Ramon Santamaria (@raysan5)
```

Comment:
You must be the one in this copyright notice, you created it!

Reply:
Alrighty, I shall add that too